src/controller.cpp

```diff
@@ -1439,10 +1439,7 @@ void controller::write_item(std::shared_ptr<rss_item> item, std::ostream& ostr)
 	textformatter txtfmt;
 	txtfmt.add_lines(lines);
 
-	unsigned int width = cfg.get_configvalue_as_int("text-width");
-	if (width == 0)
-		width = 80;
-	ostr << txtfmt.format_text_plain(width) << std::endl;
+	ostr << txtfmt.format_text_plain(80, cfg.get_configvalue_as_int("text-width")) << std::endl;
```

Comment:
This doesn't look correct to me.

Previously, we wrapped text at `text-width` if set or at 80 if `text-width` isn't set. Nonwrappable lines were left unwrapped.

Now wrappable text always wraps at 80, ignoring `text-width`. Softwrappable stuff behaves wraps at `text-width` if set, otherwise doesn't. Nonwrappable lines are left unwrapped. I believe this behaviour is wrong.

First of all, wrappable lines should definitely honour `text-width`—that's the whole point. Softwrappable, on the other hand, shouldn't be wrapped because we're outputting into a text file that have no limit to its width (and we use softwrappable for things like URLs, which are better left unmodified.)

I think you could've just leave the code as it was. Am I missing something?